src/main/java/com/coveros/selenified/Selenified.java

```diff
@@ -399,6 +399,7 @@ protected void startTest(Object[] dataProvider, Method method, ITestContext test
         desiredCapabilities.setCapability("name", testName);
         desiredCapabilities.setCapability("tags", Arrays.asList(result.getMethod().getGroups()));
         desiredCapabilities.setCapability("build", buildName);
+        capabilities.addExtraCapabilities(desiredCapabilities);     // TODO - looks like this merge isn't doing it
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1135)